examples/petsc/ex11p.cpp

```diff
@@ -0,0 +1,461 @@
+//                       MFEM Example 11 - Parallel Version
+//
+// Compile with: make ex11p
+//
+// Sample runs:  mpirun -np 4 ex11p -m ../../data/square-disc.mesh
+//               mpirun -np 4 ex11p -m ../../data/star.mesh
+//               mpirun -np 4 ex11p -m ../../data/star-mixed.mesh
```

Comment:
Could you update the sample runs to only petsc options? This is something for our test system and those are already tested in the regular run. Just have a few different options which only use petsc and one that doesn't.

Reply:
I updated the sample runs, and added a `rc_ex11p` file to test the `--slepcopts` parameter.